tests/unit/dialog/tests.html

```diff
@@ -430,6 +447,214 @@
             }, 1000);
 
         });
+
+        test("Press tab and Esc", function () {            
+            $("#dialog6").igDialog({
+                modal: true,
+                trackFocus: false
+            });
+
+            var dialog = $("#dialog6").data().igDialog;            
+
+            $.ig.TestUtil.keyInteraction(9, $("#dialog6").igDialog());
+            $.ig.TestUtil.keyInteraction(9, $("#dialog6").igDialog());
+
+            $.ig.TestUtil.keyInteraction(27,$("#dialog6").igDialog());
+
+            equal(dialog.options.state,"closed","The dialog is closed");
+
+            $("#dialog6").igDialog("option", "trackFocus", true);
+            equal(dialog.options.trackFocus, true,"The dialog's trackFocus is true");
+
+            dialog.destroy();
+        }); 
+
+        test("Restore", function () {            
+            $("#dialog6").igDialog({
+                state: "maximized"
+            });
+
+            var dialog = $("#dialog6").data().igDialog;        
+
+            dialog.restore();
+            dialog.minimize();
+            dialog.restore();    
+
+            equal(dialog.options.state,"minimized","The dialog is restored");
+
+            dialog.destroy();
+        }); 
+
+        test("Initizlize Container", function () {   
+            try 
+            {         
+                var objContainer = $("#newContainer")
+                $("#dialog5").igDialog("option", "container");
+                $("#dialog5").igDialog("option", "container", "objContainer");
+                $("#dialog5").igDialog("option", "container", objContainer);
+
+                $.ig.TestUtil.keyInteraction(9, $("#dialog5").igDialog());
+            }
+            catch (e) 
+            { 
+                ok(false, "Container not set.") 
+            }
+            ok(true,"Container is set");
```

Comment:
You can validate the container is set by getting it like so:
`ok($("#dialog5").igDialog("option", "container").is(objContainer), "Container is set");`
 Also is the try-catch needed really needed?